doc/sphinx/practical-tools/utilities.rst

```diff
@@ -364,6 +380,28 @@ line timing data:
     .. note::
        This target requires python to build the table.
 
+    .. note::
+       This target follows the same sorting order as the
+       ``print-pretty-timed-diff`` target, and supports the same
+       options for the ``TIMING_SORT_BY`` variable.
+
+    .. note::
+       By default, two lines are only considered the same if the
```

Comment:
```suggestion
       By default, two lines are matches only if
```

Reply:
Why is "are matches" better than "are considered the same"?

Reply:
Just for brevity/simplicity.  If you say something in fewer, simpler words without distorting the meaning, that's generally easier to understand.  However, I should have suggested "two lines match only if ...", which is even shorter and avoids the passive voice.

Change it or not as you wish.